kt/api-generator/src/main/kotlin/godot/codegen/generationEntry.kt

```diff
@@ -35,17 +36,18 @@ fun File.generateApiFrom(jsonSource: File, docsDir: File? = null) {
     classes.forEach { clazz ->
         clazz.properties.forEach { property ->
             val method = Method(
-                "get_${property.oldName}",
+                property.getter.convertToSnakeCase(),
                 property.type,
                 isVirtual = false,
                 hasVarargs = false,
                 arguments = listOf()
             )
             val parentClassAndMethod = tree.getMethodFromAncestor(clazz, method)
             if (parentClassAndMethod != null && !property.hasValidGetter) {
-                val parentMethodName = "get${parentClassAndMethod.first.newName}${property.newName.capitalize()}"
+                val isBoolGetter = property.getter.startsWith("is")
+                val parentMethodName = "${if (isBoolGetter) "is" else "get"}${parentClassAndMethod.first.newName}${property.newName.capitalize()}"
```

Comment:
I would separate the property name from the parent method call in the getter. Because you share this variable for both property name and parent method call the properties names are quite redundant.
Example (see in the generated files in this PR):
`isFontDistanceField` instead of `isDistanceFieldHint` inside the `Font` class. `Font` in the property name is redundant.
`isInputEventEcho` instead of `isEcho` inside the `InputEvent` class. `InputEvent` in the property name is redundant.

Reply:
This redundancy is desired, because if we do not change the name of parent getter we end up with conflicts between getter of properties and get methods from parent in api. 
Is is a particular case where this would not occur. But this happens with variable starting with get :/

Reply:
@CedNaru gave me a better idea. I'll use @JvmName (which I forgot it exists as I do not write java anymore xD)